### ISSUE TITLE
Display a list of moves during the game

### ISSUE THREAD

**Issue by galoomba1** [Infinite-Chess/infinitechess.org#342]
There should be a list of moves displayed on the side, like there is on finite chess websites. This would help greatly with finding and lining up pieces which moved far away, among other things.

**Comment by Naviary2** [comment]
@galoomba1 Are you inferring that clicking the moves would teleport you to where the move was made?

My initial thought of the benefit for adding a move list would be to help you analyze the game after it's over. But if the problem is we can't find where they moved, we can easily add a button to teleport them there, or add a setting that will auto teleport you to their move.

**Comment by galoomba1** [comment]
> Are you inferring that clicking the moves would teleport you to where the move was made?
Not necessarily. Seeing the coordinates is enough. But more features are always better, i guess.

**Comment by Naviary2** [comment]
This is honestly not a feature I have ever taken advantage of on chess.com or lichess, so I can't really appreciate it.

Those who have used a move list- what are the reasons you enjoy it? How would it improve the user experience more than just being able to rewind/forward moves with the arrow keys?

**Comment by galoomba1** [comment]
> How would it improve the user experience more than just being able to rewind/forward moves with the arrow keys?

For one, it would tell you where a far-away piece moved without having to zoom out and search for it or click a bunch of arrows.
And also I just like seeing the move list.

**Comment by Naviary2** [comment]
Yeah it would likely be a nice addition, as long as we have space for it, and we have a way to reveal the whole move if the coordinates are chopped off.
